tensorflow_probability/python/positive_semidefinite_kernels/rational_quadratic.py

```diff
@@ -116,7 +116,7 @@ def __init__(
           amplitude, scale_mixture_rate, length_scale], tf.float32)
 
       if amplitude is not None:
```

Comment:
```suggestion
      if amplitude is not None:
       amplitude = tf.convert_to_tensor(amplitude, name="amplitude", dtype=dtype)
     self._amplitude = _validate_args_if_not_none(amplitude, tf.assert_positive, validate_args)
```

Same here and below. That should fix the issue of it is missing or not.